### PR TITLE
Created Car and CarRunner classes

### PR DESCRIPTION
**Pull request by biliachenko**

### CODE REVIEW

**Comment by ozhadaie** [comment]
[.gitignore example](https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore)

**Comment by ozhadaie** [comment]
@imystique **REVIEW**

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea/**
+
+# User-specific
+.main/main.iml
```
Comment (ozhadaie) [inline .gitignore:27]: .main ? -> main/main.iml

**Review by VladShevchenko3 — APPROVED** [review]
Good job)

**Review by imystique — APPROVED** [review]

**Pull request merged**